modules/api/current.js

```diff
@@ -431,6 +431,37 @@ module.exports = {
           level: 'ERROR',
           message: stripAnsi(testStepResult.message),
         });
+
+        const hasBrowser = 'browser' in global;
```

Comment:
Is this additional check for `browser` required?

Reply:
May be better to rename it to `isBrowserPresent` or `isBrowserAvailable`? Because in this case it's sound like `is this test case has a browser`.